aderyn_core/src/framework/foundry.rs

```diff
@@ -60,13 +60,19 @@ pub fn load_foundry(foundry_root: &PathBuf) -> Result<LoadedFoundry, Box<dyn Err
         );
 
         // Run `forge build --ast` in the root
-        let _output = std::process::Command::new("forge")
+        let output = std::process::Command::new("forge")
             .arg("build")
             .arg("--ast")
             .current_dir(&foundry_root_absolute)
             .stdout(Stdio::inherit()) // This will stream the stdout
             .stderr(Stdio::inherit())
-            .status();
+            .status()
+            .expect("Failed to run `forge build --ast`");
```

Comment:
This may not be the only error. It may be that the contracts fail to compile also. We should account for all errors in this command.

Reply:
@alexroan But we already handle that by checking if the exit code is success or not in the next few lines

Reply:
This particular line only panics if forge is not available as a command in the env

Reply:
You are correct, apologies!